src/components/RoarDataTable.vue

```diff
@@ -114,14 +114,15 @@
                   toolTipByHeader(col.header).length > 0
                     ? 'text-decoration: underline dotted #0000CD; text-underline-offset: 3px'
                     : null,
+                  col.header === 'Letter Names and Sounds' ? 'width: 7em; text-wrap: wrap' : '',
```

Comment:
I believe this is no longer necessary as we've since reduced the header for ROAR-letter from "Letter Names and Sounds" to just "Letter"
```suggestion
```